src/ucp/proto/proto_am.inl

```diff
@@ -596,11 +596,13 @@ ucp_proto_get_short_max(const ucp_request_t *req,
 
 static UCS_F_ALWAYS_INLINE int
 ucp_proto_is_inline(ucp_ep_h ep, const ucp_memtype_thresh_t *max_eager_short,
-                    ssize_t length)
+                    ssize_t length, const ucp_request_param_t *param)
 {
     return (ucs_likely(length <= max_eager_short->memtype_off) ||
             (length <= max_eager_short->memtype_on &&
-             ucs_memtype_cache_is_empty()));
+             (ucs_memtype_cache_is_empty() ||
```

Comment:
it's not your code, but can you pelase fix coding style for line above
`length <= max_eager_short->memtype_on` -> `(length <= max_eager_short->memtype_on)`

Reply:
Done.